test/libsolidity/StandardCompiler.cpp

```diff
@@ -57,6 +57,9 @@ bool containsError(Json::Value const& _compilerResult, string const& _type, stri
 	return false;
 }
 
+// Note that there is a warning when compilng pre-release code which
```

Comment:
```suggestion
// Note that there is a warning when compiling pre-release code which
```

But I don't think we need or should have this comment here anyways (especially with a "This is perhaps why" guess in it ;-)). If it's true that this is only used for ignoring pre-release warnings, then we should create a separate issue for changing this to filter those warnings out and require no warnings otherwise in the tests - if there are other warnings we should either fix them or actually check what they are.

Reply:
Comment should be removed in 38ea2bbbd  This aspect confused me, but I we can deal with that some other time and way.